src/controller/SetUpCodePairer.h

```diff
@@ -180,7 +196,8 @@ class DLL_EXPORT SetUpCodePairer : public DevicePairingDelegate
     // Queue of things we have discovered but not tried connecting to yet.  The
     // general discovery/pairing process will terminate once this queue is empty
     // and all the booleans in mWaitingForDiscovery are false.
-    std::queue<RendezvousParameters> mDiscoveredParameters;
+    std::queue<SetUpCodePairerParameters> mDiscoveredParameters;
+    SetUpCodePairerParameters mCurrentParameters;
```

Comment:
Please document what mCurrentParameters is and when it has valid values... and how one can tell whether it has a valid value.